ui/raidboss/data/02-arr/trial/shiva-hm.js

```diff
@@ -49,8 +49,124 @@
       infoText: function(data, matches) {
         return {
           en: 'Free ' + data.ShortName(matches.target),
+          de: 'Befreie ' + data.ShortName(matches.target),
         };
       },
     },
   ],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Ice Soldier': 'Eissoldat',
+        'Shiva': 'Shiva',
+      },
+      'replaceText': {
+        '--adds targetable--': '--Adds erscheinen--',
+        '--frozen--': '--eingefroren--',
+        'Absolute Zero': 'Absoluter Nullpunkt',
+        'Diamond Dust': 'Diamantenstaub',
+        'Dreams Of Ice': 'Eisige Träume',
+        'Frost Blade': 'Frostklinge',
+        'Frost Staff': 'Froststab',
+        'Glacier Bash': 'Gletscherlauf',
+        'Hailstorm': 'Hagelsturm',
+        'Heavenly Strike': 'Himmlischer Schlag',
+        'Icebrand': 'Eisbrand',
+        'Icicle Impact': 'Eiszapfen-Schlag',
+        'Melt': 'Schmelzen',
+        'Permafrost': 'Permafrost',
+      },
+    },
+    {
+      'locale': 'fr',
+      'missingTranslations': true,
+      'replaceSync': {
+        'Ice Soldier': 'soldat de glace',
+        'Shiva': 'Shiva',
+      },
+      'replaceText': {
+        '\\(circle\\)': '(Kreis)',
+        '\\(cross\\)': '(Kreuz)',
```

Comment:
'\\(circle\\)': '(cercle)',
'\\(cross\\)': '(croix)',
Thank you

Reply:
could you also translate these 2 for me: 
        '--adds targetable--': '',
        '--frozen--': '',

I accidently added the cross/circle from german to the FR section

Reply:
`'--adds targetable--': '--adds ciblable--',`
`'--frozen--': '--gelé--',`
Thank you.

Reply:
done